src/libs/Navigation/CustomActions.js

```diff
@@ -6,40 +6,27 @@ import lodashGet from 'lodash/get';
 import linkingConfig from './linkingConfig';
 import navigationRef from './navigationRef';
 
+/**
+ * @returns {Object}
+ */
+function getActiveState() {
```

Comment:
Thanks rajat. Just wanted to add to remember to always test your PR before putting it out for review. If you test the current code on ios you'll see,
![image](https://user-images.githubusercontent.com/14356145/153956836-d9392a82-df31-40f4-8234-285cdcaf41ff.png)

which would quickly guide you to remove the extra method. Additionally, if you need more time you can always create a draft PR first without any assignees such that once its ready for review you can click on the ready for review button and it will then auto assign the reviewers.